packages/shylu/shylu_dd/bddc/src/setupTest.h

```diff
@@ -0,0 +1,667 @@
+
+//@HEADER
+// ************************************************************************
+// 
+//               ShyLU: Hybrid preconditioner package
+//                 Copyright 2012 Sandia Corporation
+// 
+// Under the terms of Contract DE-AC04-94AL85000 with Sandia Corporation,
+// the U.S. Government retains certain rights in this software.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are
+// met:
+//
+// 1. Redistributions of source code must retain the above copyright
+// notice, this list of conditions and the following disclaimer.
+//
+// 2. Redistributions in binary form must reproduce the above copyright
+// notice, this list of conditions and the following disclaimer in the
+// documentation and/or other materials provided with the distribution.
+//
+// 3. Neither the name of the Corporation nor the names of the
+// contributors may be used to endorse or promote products derived from
+// this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY SANDIA CORPORATION "AS IS" AND ANY
+// EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+// PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL SANDIA CORPORATION OR THE
+// CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+// PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+// LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+// NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+// SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// Questions? Contact Michael A. Heroux (maherou@sandia.gov) 
+// 
+// ************************************************************************
+//@HEADER
+
+#ifndef SETUPTESTBDDC_H
+#define SETUPTESTBDDC_H
+#include <gtest/gtest.h>
+
+#include <mpi.h>
+#include "ProblemMakerBDDC.h"
+#include "OperatorStandard.h"
+#include "ShyLU_DDBDDC_config.h"
+#include "shylu_PreconditionerBDDC.h"
+#include "shylu_KrylovSolver.h"
+#include "shylu_UtilBDDC.h"
+#include "shylu_enumsBDDC.h"
+
+#if defined(_OPENMP)
+#include <omp.h>
+#else
+#define omp_get_nested() 0
+#endif
+
+using Teuchos::RCP;
+
+namespace bddc {
+
+template <class LO, class GO, class SX, class SM> class setupTest
```

Comment:
It looks like this is a class used only for testing.  If so, it really should not be installed with the library, and should go in the test directory.

Reply:
This file and another one were moved back to the test directory as suggested.